src/metabase/lib/card.cljc

```diff
@@ -42,8 +42,12 @@
                                  (sequential? result-metadata) result-metadata))]
       (mapv (fn [col]
               (merge
+               {:base-type :type/*, :lib/type :metadata/field}
                (when-let [field-id (:id col)]
-                 (lib.metadata/field metadata-providerable field-id))
+                 (try
+                   (lib.metadata/field metadata-providerable field-id)
+                   (catch #?(:clj Throwable :cljs js/Error) _
```

Comment:
I believe `(:clj Throwable :cljs :default)` is the preferred form here as js can throw non Errors.